src/hooks/exportTemplate/useExportTemplate.ts

```diff
@@ -671,9 +687,12 @@ export default function useExportTemplate() {
                         })
                     )
                     index++
-                    row.eachCell({ includeEmpty: true }, (cell: any) => {
-                        cell.protection = { locked: false };
-                    });
+                    headers.forEach((vals: any, index: number) => {
+                        if (!headersToHide.includes(vals.key) && vals.label !== "System ID") {
```

Comment:
Im not sure that this name will be the same in all instances.
Suggestion: 
- We can map all unique attributes and keep them protected.